File: DKPython/functions/dk_warning.py
import sys 
sys.path.append(".")
from DK import *

from dk_log import *

##################################################################################
# dk_warning(<message>)
#
#    Print a warning message to the console
#
#    @message - The message to print
#
def dk_warning(*args):
#    dk_call dk_debugFunc 1

    dk_log(WARNING, args[0])





###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
def DKTEST():
#    dk_call dk_debugFunc 0

    dk_warning("test dk_warning message")

File: DKJavascript/functions/dk_warning.js
// shebang
//#####################################################################################################
//file:///C:/Users/Administrator/DigitalKnob/Development/DKJavascript/functions/dk_fileContains.js
if(typeof ActiveXObject === "function"){
	if(typeof wscript_shell !== "object")	{ var wscript_shell = new ActiveXObject("WScript.Shell"); }
	if(typeof ENV !== "object")				{ var ENV = wscript_shell.Environment("Process"); }
	if(typeof ENV === "object" && typeof wscript_shell === "object" && ENV("DKINIT_js") === "") {	
		ENV("DKINIT_js") = WScript.ScriptFullName;
		wscript_shell.Run("cmd /k mshta.exe \"file:///C:/Users/Administrator/DigitalKnob/Development/DKHta/functions/DK.hta\" | for /f \"delims=\" %a in ('findstr \"^\"') do 	@echo %a", 1, 1);
		throw new Error('Program Terminated');
	}
}
//#####################################################################################################

dk_source(DKJAVASCRIPT_DIR+"/functions/dk_log.js");

//################################################################################
//# dk_warning(msg)
//#
//#     Print an warning message to the console
//#
//#     @msg    - The message to print
//#
dk_warning = function dk_warning_f(){
	dk_log("WARNING", arguments[0]);
}





//###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
DKTEST = function DKTEST_f(){
	//dk_debugFunc(0);
	
	dk_warning("test dk_warning WARNING message");
}
